vars/buildPlugin.groovy

```diff
@@ -187,18 +195,18 @@ def call(Map addonParams = [:])
 												sshTransfer(
 													execCommand: """\
 mkdir -p /home/git/addons-binary/${versionFolder}
-chmod 444 upload/${addon}+${platform}/${addon}-*.zip
-(mv upload/${addon}+${platform}/ /home/git/addons-binary/${versionFolder}/ || cp upload/${addon}+${platform}/${addon}-*.zip /home/git/addons-binary/${versionFolder}/${addon}+${platform}/) 2> /dev/null
+chmod 444 upload/${archiveName}+${platform}/${archiveName}-*.zip
+(mv upload/${archiveName}+${platform}/ /home/git/addons-binary/${versionFolder}/ || cp upload/${archiveName}+${platform}/${archiveName}-*.zip /home/git/addons-binary/${versionFolder}/${archiveName}+${platform}/) 2> /dev/null
```

Comment:
This won't work always as for multi add-ons for some artifacts the move commands could fail and for some the copy command.

Reply:
Have you a alternative? The one I brought on begin maybe?:
```*.groovy
                            execCommand: """\
RET_VALUE=0
mkdir -p /home/git/addons-binary/${versionFolder}
for addonDir in $(ls -d upload/${variousInOnePrefix}+${platform})
do
  addonFolder=$(echo \${addonDir} | awk '{split($0,a,"/"); print a[2]}')
  chmod 444 \${addonDir}/${variousInOnePrefix}.zip
  (mv \${addonDir}/ /home/git/addons-binary/${versionFolder}/ || \
   cp \${addonDir}/${variousInOnePrefix}-*.zip /home/git/addons-binary/${versionFolder}/\${addonFolder}/) 2> /dev/null
  PUBLISHED=\$?
  if [ \$PUBLISHED -ne 0 ]; then
    echo `ls upload/\${addonFolder}/${variousInOnePrefix}-*.zip | cut -d / -f 2-` was already published >&2
    RET_VALUE=\$PUBLISHED
  fi
  rm -fr \${addonDir}/ 2> /dev/null
done
exit \$
```
But need then again a `if`, `else`

Reply:
Have added this way on second commit, is tested and runs well with the normal addon.